otterdog/eclipse-aaspe.jsonnet

```diff
@@ -99,5 +99,18 @@ orgs.newOrg('eclipse-aaspe') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('aaspe-common') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "aaspe common components",
+      has_wiki: false,
+      secret_scanning: "disabled",
```

Comment:
secret_scanning and push_protection should ideally be enabled for all repos.

See also https://otterdog.eclipse.org/projects/dt.aaspe for information about your current status and configuration.

Reply:
thanks, will amend the config